tests/datavolume_test.go

```diff
@@ -128,6 +126,73 @@ var _ = Describe("DataVolume tests", func() {
 		},
 			table.Entry("succeed when given a source PVC with a data", fillCommand, cdiv1.Succeeded, "dv-clone-test-1"),
 		)
+
+		table.DescribeTable("with import source = \"none\" should", func(phase cdiv1.DataVolumePhase, dataVolumeName string, eventReason string) {
+			dataVolume := utils.NewDataVolumeForBlankRawImage(dataVolumeName, "1Gi")
+
+			By(fmt.Sprintf("creating new datavolume %s", dataVolume.Name))
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+			Expect(err).ToNot(HaveOccurred())
+
+			By(fmt.Sprintf("waiting for datavolume to match phase %s", string(phase)))
+			utils.WaitForDataVolumePhase(f.CdiClient, f.Namespace.Name, phase, dataVolume.Name)
+
+			// verify PVC was created
+			By("verifying pvc was created")
+			_, err = f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(dataVolume.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			By(fmt.Sprint("Verifying event occurred"))
+			Eventually(func() bool {
+				events, err := RunKubectlCommand(f, "get", "events", "-n", dataVolume.Namespace)
+				if err == nil {
+					fmt.Fprintf(GinkgoWriter, "%s", events)
+					return strings.Contains(events, eventReason)
+				}
+				fmt.Fprintf(GinkgoWriter, "ERROR: %s\n", err.Error())
+				return false
+			}, timeout, pollingInterval).Should(BeTrue())
+
+			err = utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolume.Name)
+			Expect(err).ToNot(HaveOccurred())
+
+		},
+			table.Entry("succeed creaing blank-image dv", cdiv1.Succeeded, "blank-image-dv", controller.ImportSucceeded),
```

Comment:
looks like these tests (new and existing) are very similar.  It is also kind of weird that they are table driven tests that have only one Entry.  Does it make sense to combine them into a single table driven test with multiple Entrys?

Reply:
done